_data/guides.yaml

```diff
@@ -34,6 +34,9 @@ categories:
       - title: Securing WildFly Apps with SAML on OpenShift
         url: /guides/security-saml-openshift
         description: Learn how to secure applications deployed to WildFly on OpenShift with SAML.
+      - title: Securing a WildFly Application Using OpenID Connect With Additional Scope Values
```

Comment:
It would be good to group the OIDC guides together. Let's put this one in between the identity propagation and the management console OIDC guides.

Reply:
fixed!